src/kiwixapp.h

```diff
@@ -45,6 +45,8 @@ class KiwixApp : public QtSingleApplication
         ZoomInAction,
         ZoomOutAction,
         ZoomResetAction,
+        HistoryBackAction,
+        HistoryForwardAction,
```

Comment:
You declare a `HistoryForwardAction` but never define it.

You never use `HistoryBackAction` and `HistoryForwardAction`.